core/include/usGetBundleContext.h

```diff
@@ -28,14 +28,15 @@
 #error Missing preprocessor define US_BUNDLE_NAME
 #endif
 
+#include <usBundleContext.h>
+#include <usBundleContextPrivate.h>
```

Comment:
This will affect issue #43 requiring that ```usBundleContextPrivate.h``` be a "public" header. I would like to avoid doing this. 
Is there anything that can be done? Perhaps moving the declaration of ```MakeBundleContext```?